fvtest/porttest/si.cpp

```diff
@@ -2403,6 +2709,55 @@ TEST(PortSysinfoTest, sysinfo_cgroup_get_memlimit)
 	return;
 }
 
+/**
+ * Test omrsysinfo_get_cgroup_subsystem_list.
+ */
+TEST_F(CgroupTest, sysinfo_get_cgroup_subsystem_list)
+{
+	OMRPORT_ACCESS_FROM_OMRPORT(portTestEnv->getPortLibrary());
+	const char *testName = "omrsysinfo_get_cgroup_subsystem_list";
+
+	reportTestEntry(OMRPORTLIB, testName);
+
+#if defined(LINUX)
+	OMRCgroupEntry *entries = omrsysinfo_get_cgroup_subsystem_list();
+	OMRCgroupEntry *temp = entries;
+
+	if (NULL == temp) {
+		EXPECT_EQ(CgroupTest::available, (uint64_t)0);
+	} else {
+		do {
+			EXPECT_EQ(CgroupTest::available & temp->flag, temp->flag);
+			switch(temp->flag) {
+			case OMR_CGROUP_SUBSYSTEM_CPU:
+				EXPECT_EQ(temp->cgroup, cpuCgroup);
+				EXPECT_STREQ(temp->subsystem, "cpu");
+				break;
+			case OMR_CGROUP_SUBSYSTEM_MEMORY:
+				EXPECT_EQ(temp->cgroup, memCgroup);
+				EXPECT_STREQ(temp->subsystem, "memory");
+				break;
+			case OMR_CGROUP_SUBSYSTEM_CPUSET:
+				EXPECT_EQ(temp->cgroup, cpusetCgroup);
+				EXPECT_STREQ(temp->subsystem, "cpuset");
+				break;
+			default:
+				FAIL() << "Unsupported subsystem";
+			}
+			temp = temp->next;
+		} while (temp != entries);
+	}
+#else /* defined(LINUX) */
+	if (NULL != omrsysinfo_get_cgroup_subsystem_list()) {
+		outputErrorMessage(PORTTEST_ERROR_ARGS, "omrsysinfo_get_cgroup_subsystem_list returned not null on non-Linux\n");
+	}
+#endif /* defined(LINUX) */
+
+	reportTestExit(OMRPORTLIB, testName);
+	return;
+}
+#endif /* !defined(LINUX) || (GTEST_GCC_VER_ >= 40900) */
```

Comment:
We should send a message to the OMR user that the Cgroup tests are disabled. Once all the machines in the OMR build-farm have newer compilers, we should attempt to change this to an `#error`.
```suggestion
#else /* !defined(LINUX) || (GTEST_GCC_VER_ >= 40900) */
#warning "Cgroup tests are disabled due to an unsupported compiler."
#endif /* !defined(LINUX) || (GTEST_GCC_VER_ >= 40900) */
```

Reply:
Works for me locally but may not be supported on older gcc. Pushed the changes, can you try running the builds?

Reply:
Not sure what happened with the windows build but the ppc failures also appear in other PRs